overlay/acm/acm-2.14/kustomization.yaml

```diff
@@ -32,3 +32,21 @@ patches:
       kind: Component
       name: siteconfig.*
     path: siteconfig-branch-patch.yaml
+  - target:
+      kind: ReleasePlan
+    path: releaseplan-dev.yaml
+  - target:
+      kind: ReleasePlanAdmission
+    path: rpa-dev.yaml
```

Comment:
May need to have a conversation because you might just be calling this a different way then I'm used to but I don't these would run since there's nothing to patch. It might be better to just apply them as resources when we want to call them.